client/fingerprint_manager.go

```diff
@@ -122,42 +122,40 @@ func (fm *FingerprintManager) Reload() {
 // setupFingerprints is used to fingerprint the node to see if these attributes are
 // supported
 func (fm *FingerprintManager) setupFingerprinters(fingerprints []string) error {
-	var appliedFingerprints []string
-
 	for _, name := range fingerprints {
 		f, err := fingerprint.NewFingerprint(name, fm.logger)
-
 		if err != nil {
 			fm.logger.Error("error fingerprinting", "error", err, "fingerprinter", name)
 			return err
 		}
 
-		detected, err := fm.fingerprint(name, f)
```

Comment:
the underlying fingerprinting `err` here is no longer bubbling up in the new implementation, but only logged as a debug line - isn't that a behavior change?

Reply:
It is a change in behavior. As @tgross noted above, previously the failure of any single fingerprinter was considered fatal. With this change that is no longer the case. I'll think on this some.